### DISCUSSION TITLE
GrandOrgue registri

### DISCUSSION THREAD

**Discussion by marconatale00** [GrandOrgue/grandorgue#1092] [Developers]
Good evening I wanted to pose a problem I just built an organ with the vob when I start it starts with an eternity of yellow triangles with an exclamation point and after 4 minutes I load the organ I ask if it is possible to remove all those '' errors "because it takes longer time there that has not loaded the organ an idea that I give you make appear only the real errors

**Comment by paolo2504** [comment]
ciao Marco, guarda che non so se risponderanno mai a un post in italiano....
Riscrivilo in inglese...
Nel frattempo ti posso dire che i warnings in genere non possono essere eliminati ma in ogni caso non impedisono il corretto caricamento. Solo se sono fatal (in rosso) bisogna risolvere il problema.
Che tipo di errori è ? Allega un esempio..

**Comment by marconatale00** [comment]
Hi Paul thanks for the answer I'll explain how I can why
I'm away from home, finished building the 3-release organ by putting the
samples I start it and a screen opens with all of the messages with a
yellow triangle with an exclamation mark with ex. ⚠️Warning: Name
of the file I Manual \ Mixtur \ A0 \ 093-a
⚠️Warning: Name of the file I Manual \ Mixtur \ R0 \ 093-a
⚠️Warning: Name of the file I Manual \ Mixtur \ R1 \ 093-a
⚠️Warning: Name of the file I Manual \ Mixtur \ R2 \ 093-a
This is an example but it takes longer to load all this wording
that organ itself, I was wondering if you can eliminate all these errors
because it takes so long. Thank you, see you soon

**Comment by eturpault** [comment]
Hi @marconatale00, the warnings should be solved rather in the ODF than by modifying the GrandOrgue SW. Could you share the ODF (file .organ) which is causing these warnings ? There might be some errors in the files naming.

**Reply by marconatale00** [reply]
Hi @eturpault, of course I am attaching the code the organ was created with the appropriate software

**Reply by marconatale00** [reply]
[Organo.txt](https://github.com/GrandOrgue/grandorgue/files/8554634/Organo.txt)

**Comment by eturpault** [comment]
Looking at this ODF, in order to remove these warning you could try to replace in the folders names the blank spaces by underscore characters. In the source code of GO I don't see what can generate this warning.
For example rename "I Manuale" in "I_Manuale", "II Manuale" in "II_Manuale", "III Manuale" in "III_Manuale", and do the same renaming in the ODF.

I see that there are some definition issues in this ODF :
- in Label001, ... , Label007 : the attribute Image is written with a lower-case letter i : image= must be rather Image=
- in Manual000 : the attributes ImageOff_LastC and ImageOn_LastC are defined twice
- in Manual000, ... , Manual004 : the attribute NumberOfTremulants is set at "0.." (NumberOfTremulants=0.. must be rather NumberOfTremulants=0)
- in Stop001, ... , Stop043 : the row IDefaultToEngaged=N is not correctly spelled and must be removed, The attribute DefaultToEngaged=N is already defined elsewhere in the section
- in Coupler001, ... , Coupler018 : the attribute Comment is useless (not defined in the ODF syntax)

**Reply by eturpault** [reply]
You can use the following Python tool to edit/check your ODF : https://github.com/GrandOrgue/grandorgue/discussions/917#discussioncomment-2576444.
It runs with Python3, or there is the executable for Windows and Linux.
